package/files/rmt-cli_bash-completion.sh

```diff
@@ -56,7 +56,7 @@ _rmt-cli_products()
 
 	current_word=${COMP_WORDS[COMP_CWORD]}
 	options="list enable disable help"
-	flags="--all --csv --release-stage="
+	flags="--all --csv --release-stage= --name --version --arch"
```

Comment:
Since all three new flags need user input, I think you should append a `=` to each of the new options, as we have for `--release-stage=`.

Reply:
Done. Question, my thinking was that the bash completion would be different than the cli `help` and use without completion, 
is that oik? Same for `--release-stage` flag

Reply:
Hm, I just tested with your latest changes:
```
rmt-cli products list --arch=
No matching products found in the database.
Only enabled products are shown by default. Use the '--all' option to see all products.
```

So your code is robust in enough to handle this, I'd say. What happens is the products are filtered to only show those with an arch value of empty string. So no products are shown. I think that is intuitive enough :)

And with your improved bash completion that, when hitting <kbd>TAB</kbd><kbd>TAB</kbd> after typing `rmt-cli products list --`, will suggest `--arch= --name= --version=` I'm sure our users will get it, and only "break" it when that's what they want to do ;)

Great job! I'll ask someone else from the core team for a second pair of eyes over your changes but I think we are good to go.